gnome-initial-setup/gis-driver.c

```diff
@@ -178,6 +179,11 @@ gis_driver_set_user_language (GisDriver *driver, const gchar *lang_id)
   GisDriverPrivate *priv = gis_driver_get_instance_private (driver);
   g_free (priv->lang_id);
   priv->lang_id = g_strdup (lang_id);
+
+  /* Now, if we already have a user configured, make sure to
+   * propogate that change to the user */
+  if (priv->user_account)
+    act_user_set_language (priv->user_account, lang_id);
```

Comment:
I think this may have been related to the old code I had in place when autoconfiguring the FBE on subsequent invocations. I left it in for now since I wasn't entirely sure whether that was still used, I'll follow up.